src/main/java/ch/njol/skript/conditions/CondIsOp.java

```diff
@@ -0,0 +1,50 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.conditions;
+
+import org.bukkit.OfflinePlayer;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+
+@Name("Is Op")
+@Description("Check whether a player is an operator.")
+@Examples({"if player is op:", "if all players are not ops:"})
+@Since("INSERT VERSION")
+public class CondIsOp  extends PropertyCondition<OfflinePlayer> {
+	
+	static {
+		register(CondIsOp.class, PropertyType.BE, "[an] op[erator][s]", "offlineplayers");
```

Comment:
Maybe you could do ``[a[n]] op[erator][s]`` to prevent people accidentally using ``a`` instead of ``an`` and getting confused

Reply:
I'd rather not. 
This is meant as a fix for people incorrectly using the op entity type, as you can see in the lang file for that
```
	op:
		name: op¦s @an
		pattern: op(|1¦s)
```
it uses an not a, as this is proper grammar. 
I don't think we should start spelling things wrong in Skript just to make it easier on people making mistakes.